contentcuration/contentcuration/viewsets/channelset.py

```diff
@@ -78,6 +78,7 @@ def get_queryset(self):
         queryset = super(ChannelSetViewSet, self).get_queryset()
         user_id = not self.request.user.is_anonymous and self.request.user.id
         edit = Exists(User.channel_sets.through.objects.filter(user_id=user_id, channelset_id=OuterRef("id")))
+        queryset = queryset.filter(edit=True)
```

Comment:
The changes look correct to me! Thanks @GarvitSinghal47! Just one suggestion was that we could also create the query set directly in one go, which seems to be bit more readable for developers.
`        queryset = super(ChannelSetViewSet, self).get_queryset().annotate(
            edit=Exists(
                User.channel_sets.through.objects.filter(
                    user_id=user_id,
                    channelset_id=OuterRef("id")
                )
            )
        ).filter(edit=True)`